### DISCUSSION TITLE
error while deploying

### DISCUSSION THREAD

**Discussion by Elmahallawy** [alshedivat/al-folio#2537] [Q&A] [answered]
I get the following when I deploy: 
NbConvertApp] Converting notebook C:/Users/yasmi/AppData/Local/Temp/jekyll-jupyter-notebook20240702-4072-11kphtk.ipynb to html
                    done in 158.928 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\mdb.min.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\main.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\jupyter.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\jupyter-monokai.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\jupyter-grade3.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\jekyll-pygments-themes-native.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\jekyll-pygments-themes-github.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\bootstrap.min.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\bootstrap-toc.min.css'
ENOENT: no such file or directory, open 'C:\Users\yasmi\desktop\Elmahallawy.github.io\_site\assets\css\_site\assets\css\academicons.min.css'

The live website has no style while locally it appears well: 
![image](https://github.com/alshedivat/al-folio/assets/87029356/b848867a-c77c-4d9f-b9a9-88a1fc09579f)

**Comment by george-gca** [comment]
You should [follow the recommended approach](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach) to the line. Only after the website is running you further modify it.

Duplicate of #2436, #2227.

**Reply by Elmahallawy** [reply]
The website is running locally. But the CSS is not working live.
I followed the mentioned steps.

**Reply by Elmahallawy** [reply]
Also I can't find the _sites

**Reply by george-gca** [reply, marked as answer]
If you follow the recommended steps, you'll only do 2 changes in `_config.yml` (among with other configuration steps) and the site should be live and working. I recommend you start over.

The `_sites/` is never uploaded to the repository since it is always generated from build. That is handled automatically in `.github/workflows/deploy.yml`.

**Reply by Elmahallawy** [reply]
Thank you so much
